### PR TITLE
Visual updates to user dropdown and 'My sites' header

### PR DESCRIPTION
**Pull request by shornuk**
### Changes

These are just niggles I've had when viewing the site. Not sure if you want things like this being submitted so feel free to stop me :) 

Updated the 'My sites' bar as it was getting squashed on mobile. Add a touch of padding to the name dropdown to even the hover state up.

See here...
Artboard

Below you'll find a checklist. For each item on the list, check one option and delete the other.

### Tests
- [ ] Automated tests have been added

### Changelog
- [ ] Entry has been added to changelog

### Documentation
- [ ] This change does not need a documentation update

### CODE REVIEW

**Comment by metmarkosaric** [comment]
thanks @shornuk! got this exact report on Twitter today so this is very timely :) I'll let Uku check but looks good to me!

File: CHANGELOG.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ All notable changes to this project will be documented in this file.
 - Adds support for single-day date selection plausible/analytics#495
 - Goal conversion rate in realtime view is now accurate plausible/analytics#500
 - Various UI/UX issues plausible/analytics#503
+- Various visual css tweaks
```
Comment (ukutaht) [inline CHANGELOG.md:45]: No need to add a note in Changelog about such a small change

Comment (shornuk) [reply CHANGELOG.md:45]: Thought it was probably unnecessary, but wanted to follow the checklist 👍

Comment (ukutaht) [reply CHANGELOG.md:45]: Will merge this as soon as the changelog message is removed :)

Comment (shornuk) [reply CHANGELOG.md]: Nice. Just done now.

**Comment by ukutaht** [comment]
Thanks!

**Pull request merged**